exaudfclient/base/javacontainer/BUILD

```diff
@@ -3,38 +3,48 @@ package(default_visibility = ["//visibility:public"])
 genrule(
     name = "exascript_java_tmp_cc",
     cmd = """
-        mkdir -p $(location java_src)/com/exasol/swig        
+        mkdir -p java_src/com/exasol/swig
         mkdir -p build_exascript_java_tmp_cc/exaudflib
         cp "$(location //exaudflib:swig/script_data_transfer_objects_wrapper.h)" "$(location //exaudflib:exascript.i)" build_exascript_java_tmp_cc/exaudflib
-        cd build_exascript_java_tmp_cc     
-        swig -v -O -DEXTERNAL_PROCESS -Wall -c++ -java -addextern -module exascript_java -package com.exasol.swig -outdir "../$(location java_src)/com/exasol/swig" -o "../$(location exascript_java_tmp.cc)" exaudflib/exascript.i
-        """,
-    outs = ["exascript_java_tmp.cc","java_src"],
+        cd build_exascript_java_tmp_cc
+        swig -v -O -DEXTERNAL_PROCESS -Wall -c++ -java -addextern -module exascript_java -package com.exasol.swig -outdir "../java_src/com/exasol/swig" -o "../exascript_java_tmp.cc" exaudflib/exascript.i
+        cd ..
+        tar cf java_src.tar ./java_src
```

Comment:
return java files directly, if they are not needed to often

Reply:
Done